test/include.test.js

```diff
@@ -33,14 +35,14 @@ describe('include', function() {
           should.not.exist(owner);
         } else {
           should.exist(owner);
-          owner.id.should.eql(p.ownerId);
+          should.equal(owner.id, p.ownerId);
```

Comment:
IIRC, `equal` is `===` while `eql` is deep comparision. See http://shouldjs.github.io/#assertion-equal